packages/metal-dom/src/keyboardEvents.js

```diff
@@ -0,0 +1,156 @@
+'use strict';
+
+import { object } from 'metal';
+
+export const KEYMAP = {
+  'BACKSPACE': 8,
```

Comment:
Sorry @mairatma and @fernandosouza, I have to take back what I said. Since we’re talking about `keyup:KEY` events, you need to have the whole thing at runtime. Rollup couldn’t help here.

This doesn’t mean that having the source exporting stuff like that is good anyway. You can later `import * as keys from '../keys'` to access them as a single object.